user_info.go

```diff
@@ -30,3 +30,11 @@ func (c *WorkwxApp) ListUsersByDeptID(deptID int64, fetchChild bool) ([]*UserInf
 	}
 	return users, nil
 }
+
+// UseridByMobile 通过手机号获取 userid
+func (c *WorkwxApp) UseridByMobile(mobile string) (string, error) {
```

Comment:
Function name must start with a verb. Also `userid` is consistently `UserID` in this library's public API.

Reply:
fixed